### ISSUE TITLE
looks like tesla have changed battery data

### ISSUE THREAD

**Issue by doubledrat** [tdorssers/TeslaPy#161]
as of last night, it seems that tesla is not returning the following battery data (in the UK anyway)

total_pack_energy
energy_left

I've tested the new fleet API response, and the information is not in there either

*sigh*

there is "nameplate_energy" and "percent_charged", so you can sort of calculate the missing information from there, but total_pack_energy was never the same as nameplate_energy

**Comment by timhawker** [comment]
Yep the change is the same here in New Zealand. Looks like I can no longer calculate battery SoH.
